### PR TITLE
apply_rule/apply_rules updates

### PR DESCRIPTION
**Pull request by rad-pat**
I know this is going to be a little controversial, but I've made some amends to `apply_rules`

### CODE REVIEW

**Review by rea725 — APPROVED** [review]

File: plaidcloud/utilities/frame_manager.py [hunk 1]
```diff
@@ -1260,137 +1248,147 @@ def apply_rules(df, df_rules, target_columns=['value'], include_once=True, show_
...
-        else:
-            df_subset = df[df['include'] == True]
-
-        # Populate target columns as specified in split
-        for column in target_columns:
-            df_subset[column] = rule[column]
-
-        # need to find a way to flip the flag once data has been selected
+        if iteration == '2' or iteration == '3':
+            a = 1
```
Comment (rad-pat) [inline plaidcloud/utilities/frame_manager.py:1270]: Probably shouldn't be in final code

Comment (rad-pat) [reply plaidcloud/utilities/frame_manager.py:1270]: Is iteration a string? If you had 10+ iterations, would that sort numerically, or alpha-numerically?

File: plaidcloud/utilities/frame_manager.py [hunk 2]
```diff
@@ -1260,137 +1248,147 @@ def apply_rules(df, df_rules, target_columns=['value'], include_once=True, show_
...
+                        df['log'] = list(map(add_message, df['log']))
+                    error_msg = ' (rule_num {0}) {1} error: {2}'.format(index, rule[condition_column], e)
+                    logger.exception('EXCEPTION {}'.format(error_msg))
+            else:
+                df_subset = df[df['include'] == True]
+
+            # Populate target columns as specified in split
+            for column in target_columns:
+                if column == 'VT':
+                    a = 1
```
Comment (rad-pat) [inline plaidcloud/utilities/frame_manager.py:1316]: Not required in final code

**Review by rad-pat — COMMENTED** [review]
Comments

**Pull request merged**